controller/place.js

```diff
@@ -50,6 +53,7 @@ function setup( apiConfig, esclient ){
         // only consider for status 408 (request timeout)
         if (isRequestTimeout(err) && operation.retry(err)) {
           logger.info(`request timed out on attempt ${currentAttempt}, retrying`);
+          debugLog.stopTimer(req, initialTime, 'request timed out, retrying');
```

Comment:
Wouldn't this just keep logging time deltas based on when the first request was made rather than the time that the subsequent attempts were made?

Reply:
That is, if `initialTime` is set to `0`, the first failed request takes 10ms and the second successful request takes 13ms, then 2 debug log messages would be output: `took 10ms` and `took 23 ms` even though the 2nd only took 13ms.

Reply:
You're right, all the durations would be based off the very first attempt's start time. A fix is to call a new `beginTimer()` immediately after stopping the timer during a failed request. 

```
debugLog.stopTimer(req, initialTime, 'request timed out, retrying');
initialTime = debugLog.beginTimer(req, `Attempt ${currentAttempt +1}`);
```
Would this mean overwriting the `initialTimer` constant (changing to `let`)? The call to stop timer will not know which is the "true" start time to calculate delta.

Reply:
Try changing the scope of `initialTime` so that it can be set per attempt and still use `const`.

Reply:
I tried to create an array to store all the start time values. The stopTimer will take the last element of the array, which will always be the most recent query attempt start time. 

```
const timerStarts = new Array();
timerStarts.push(debugLog.beginTimer(req));
[...]
debugLog.stopTimer(req, _.last(timerStarts), 'request timed out, retrying');
timerStarts.push(debugLog.beginTimer(req, `Attempt ${currentAttempt +1}`));
```
However the linter requires `array literal notation []`. If the array is constant, how can I push more elements to it? I chose an array over an object to preserve the order.

Reply:
How about just making `initialTime` scoped to the `operation.attempt((currentAttempt) => { ... }` function on line 49?